lib/patched/env.rb

```diff
@@ -20,14 +20,29 @@ module PatchedEnv
   ##
   # Monkey patch to support Cloud SQL as an adapter
   def db
+    db = instance_variable_get :@db
+    return db if db
```

Comment:
These two lines can now be simplified as `return @db if @db`. The set statement below can just set `@db` by using `@db=`.